src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -3254,6 +3231,13 @@ Sketcher::SketchObject *ViewProviderSketch::getSketchObject() const
     return dynamic_cast<Sketcher::SketchObject *>(pcObject);
 }
 
+void ViewProviderSketch::setAngleSnapping(bool val, Base::Vector2d ref)
+{
+    assert(snapManager);
+    snapManager->angleSnapEnabled = val;
+    snapManager->angleSnapRef = ref;

```

Comment:
And the parameter angleSnapRef.

Reply:
You mean you want the `snapManager->angleSnapRef` renamed to `snapManager->referencePoint` ?

Reply:
Yep, or any meaningful name. You know better than we do. ;l

Reply:
Both to meaningful names. 

When a developer reads the class declaration, the more meaningful information we provide the better his position to understand what we wanted. When a developer reads the function declaration/definition, the better his position to understand what we wanted.